### PR TITLE
Translate to Chinese (Mandarin) #3529

### PR DESCRIPTION
**Pull request by ghost**
Translate to Chinese (Mandarin) #3529

### CODE REVIEW

File: app/_locales/zh_CN/messages.json [hunk 1]
```diff
@@ -1,10 +1,609 @@
...
+    "message": "加载代币..."
+  },
+  "localhost": {
+    "message": "本地主机 8545"
+  },
+  "logout": {
+    "message": "登出"
+  },
+  "loose": {
+    "message": "丢失"
```
Comment (tmashuang) [inline app/_locales/zh_CN/messages.json]: This does not seem correct.

**Review by kumavis — APPROVED** [review]

**Comment by kumavis** [comment]
@Orangem21 thanks for making this happen! 😻 😻 
please see invitation to MetaMask translators team 🦊 https://github.com/MetaMask

**Comment by alacrity26** [comment]
@Orangem21 are you still working on this? 
20180423 mm kb 2ftix mandarin button suggestion 1

On the send ETH page, "NEXT" is translated as "下一个"

Could you consider "继续" instead, even though it's not a direct literal translation of next, and it means continue? I feel in that context, maybe it'd make more sense contextually. 

If not, how about "下一步"?

**Comment by Orangem21** [comment]
OK . I will fix ASAP

**Comment by alacrity26** [comment]
@Orangem21 thanks! 

I have found other screenshots where we need your help:
20180429 mm kb 2ftix 1 simplified chinese add token screen left out

20180429 mm kb 2ftix simplified chinese confirm screen left out

**Comment by Orangem21** [comment]
@alacrity26 OK, thanks.

**Pull request merged**